scripts/nexus-export.sh

```diff
@@ -54,8 +55,31 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 1000Gi
+      storage: $backupSpaceNeeded
 EOF
+elif [[ ${backupSpaceNeeded} != $(kubectl get pvc -n nexus nexus-bak -o jsonpath='{.status.capacity.storage}') ]]; then
+  echo "The backup PVC size does not match the nexus-data PVC."
+  read -p "Would you like to delete the nexus-bak PVC and recreate it to match the size? (y/n)" -n 1 delete
+  delete=${delete,,}
+  if [ "$delete" = "y" ]; then
+    kubectl -n nexus delete pvc nexus-bak
+    cat << EOF | kubectl -n nexus create -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nexus-bak
+spec:
+  storageClassName: k8s-block-replicated
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: $backupSpaceNeeded
+EOF
+  else
+    echo "Exiting script."
```

Comment:
Do you want to add a more detail here? Such as "A nexus backup cannot be taken because the the Backup PVC size does not match the nexus-data PVC. Exiting script." And should it be an exit 1 since the script did not complete what it was supposed to?